hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/StreamReadOperator.java

```diff
@@ -168,6 +174,8 @@ private void processSplits() throws IOException {
       currentSplitState = SplitState.IDLE;
     }
 
+    readMetrics.setSplitLatestCommit(split.getLatestCommit());
+
```

Comment:
Does the metrics got updated for each read?

Reply:
The metrics will be updated for every new split.

Reply:
Maybe we should move it to line 170.